pipelines/build/common/create_job_from_template.groovy

```diff
@@ -78,8 +78,10 @@ pipelineJob("$buildFolder/$JOB_NAME") {
                 <dt><strong>CODEBUILD</strong></dt><dd>Use a dynamic codebuild machine if no other machine is available</dd>
                 <dt><strong>DOCKER_IMAGE</strong></dt><dd>Use a docker build environment</dd>
                 <dt><strong>DOCKER_FILE</strong></dt><dd>Relative path to a dockerfile to be built and used on top of the DOCKER_IMAGE</dd>
+                <dt><strong>PLATFORM_CONFIG_LOCATION</strong></dt><dd>Repo owner, branch name and relative path to the platform specific configuration for this paticular OS</dd>
                 <dt><strong>CONFIGURE_ARGS</strong></dt><dd>Arguments for ./configure. Escape all speech marks used within this parameter.</dd>
                 <dt><strong>OVERRIDE_FILE_NAME_VERSION</strong></dt><dd>Set the version string on the file name</dd>
+                <dt><strong>USE_ADOPT_BASH_SCRIPTS</strong></dt><dd>Use Adopt's make-adopt-build-farm.sh script</dd>
```

Comment:
I'm wondering if that's the right name - bash scripts doesn't convey intent.  It's really build scripts no?

Reply:
Build scripts imply our groovy AND bash scripts. I put bash as it only affects the shell scripts. If you think it's unclear, I could change it to `USE_ADOPT_SHELL_SCRIPTS`?

Reply:
SHELL would be better, thanks!

Reply:
Done